src/main/java/org/traccar/web/OverrideFilter.java

```diff
@@ -49,7 +49,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             chain.doFilter(request, response);
             return;
         }
-
+        response.setCharacterEncoding("UTF-8");
```

Comment:
What if it's not a text? Does it work without it? What's the default value for you?

Reply:
> What if it's not a text? Does it work without it? What's the default value for you?

You are right. This line is unnecessary.